main/lsp/DefLocSaver.cc

```diff
@@ -47,4 +45,33 @@ unique_ptr<ast::MethodDef> DefLocSaver::postTransformMethodDef(core::Context ctx
     return methodDef;
 }
 
+unique_ptr<ast::UnresolvedIdent> DefLocSaver::postTransformUnresolvedIdent(core::Context ctx,
```

Comment:
We lose this information when building the CFG, so we have to capture it in the AST.

In the CFG, references to ivars and cvars are represented as operations on variables that alias the field symbol.

Reply:
+1, CFG is the right location for this.